packages/compiler/lib/service.ts

```diff
@@ -62,36 +62,6 @@ export function getServiceTitle(program: Program): string {
   return serviceDetails.title || "(title)";
 }
 
-export function $serviceHost(context: DecoratorContext, target: Type, host: string) {
```

Comment:
Not sure we should make the breaking change at this point.  Instaed, can we call servers with the static value provided?

Reply:
The problem with that his one is servicehost is in the compiler but here server is in the http library so we can’t just call it. We could move the servicehost to the http library temporarily but that also doesn’t seem like it is that worth it. We have plenty of breaking changes coming(versioning, emitter options) and this one is very simple to fix. We should probably just write a good migration doc between version

Reply:
Alternately, then, can we leave serviceHost and put in a deprecation warning when this decorator is used, asking users to migrate to 'server'?  As much as possible, I would like to avoid breaking changes, and allow existing customers to transition

Reply:
I guess, we could but that also means the emitters need to listen to both values. As long as we are being strict with keeping the things deprecated for a 1 version transition